### DISCUSSION TITLE
Vacation Mode

### DISCUSSION THREAD

**Discussion by SnarkyDeveloper** [monkeytypegame/monkeytype#4756] [Ideas] [answered]
Like a specific cooldown for like 10 days, where your streak never resets, this might be helpful because sometimes I can’t type since I don’t have a device on me.

**Comment by Miodec** [comment]
Not sure how i feel about it. This kinda makes the streaks less "impactful" since people can just go on vacation and never lose it.

**Reply by SnarkyDeveloper** [reply]
True , but only like a few days. Maybe 10 days a year I think is fair

**Reply by SnarkyDeveloper** [reply]
@Miodec any follow up on this?

**Reply by Miodec** [reply, marked as answer]
Sorry, i dont think it will be added.

**Reply by SnarkyDeveloper** [reply]
Got it
